src/docfx/lib/json/YamlUtility.cs

```diff
@@ -22,7 +22,12 @@ internal static class YamlUtility
 
         public static string ReadMime(TextReader reader)
```

Comment:
`ReadMime` => `ReadSchema`?

Reply:
I will leave it as it is here, we have another method `ReadSchema` reading `$schema` from JSON